tests/decode-chdlc-01/test.yaml

```diff
@@ -2,35 +2,34 @@ requires:
 
   min-version: 6.0.0
 
-
 checks:
 
-    - filter:
-        count: 1
-        match:
-            event_type: http
-            http.hostname: "view.atdmt.com"
-            http.status: 200
-            http.length: 8079
-
-    - filter:
-        count: 1
-        match:
-            event_type: fileinfo
-            fileinfo.state: CLOSED
-
-    - filter:
-        count: 1
-        match:
-            event_type: alert
-            alert.signature_id: 666
-
-    - filter:
-        count: 1
-        match:
-            event_type: flow
-            proto: TCP
-
-    - stats:
-        decoder.ipv4: 17
-        decoder.chdlc: 17
+  - filter:
+      count: 1
+      match:
+        event_type: http
+        http.hostname: "view.atdmt.com"
+        http.status: 200
+        http.length: 8079
+
+  - filter:
+      count: 1
+      match:
+        event_type: fileinfo
+        fileinfo.state: CLOSED
+
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        alert.signature_id: 666
+
+  - filter:
+      count: 1
+      match:
+        event_type: flow
+        proto: TCP
+
+  - stats:
+      decoder.ipv4: 17
+      decoder.chdlc: 17
```

Comment:
This file is not changed but for white spaces, is it ?

Reply:
Yes, the only changes were whitespace related.

Reply:
Could we keep it as before then ?